migtests/tests/oracle/clob-datatype-test/validate

```diff
@@ -71,12 +71,20 @@ def migration_completed_checks(tgt):
 	tgt.assert_all_values_of_col("clob_test", "b_data", "public", expected_values=[None] * 8)
 	print("✓ NCLOB and BLOB correctly NULL (not exported)")
 
-	# Validate large CLOBs pattern (IDs 101-103) if they exist
-	print("Validating large CLOB content pattern...")
+	# Validate large CLOBs pattern and length (IDs 101-103) if they exist
+	print("Validating large CLOB content and length...")
+	expected_large_clob_size = 200 * 1024 * 1024  # 200MB
+	tolerance = 32768  # Allow 32KB tolerance for chunking variations
```

Comment:
how are we sure this much will be buffer?

Reply:
Ran the test a bunch of times. The value is falling under this itself.
Had started with a lower value but it was failing. We can see in the nightly runs if it fails often we can increase it.